### PR TITLE
permissioned-toke updates

### PR DESCRIPTION
**Pull request by robert-zaremba**
+ use account module
+ use tesla shares instead of gold
+ update comments

### CODE REVIEW

**Review by ratik21 — APPROVED** [review]

**Review by ratik21 — CHANGES_REQUESTED** [review]
image

File: examples/permissioned-token/scripts/common/common.js [hunk 1]
```diff
@@ -10,23 +10,30 @@ exports.executeTransaction = async function (deployer, txnParams) {
...
+    account.push({
       type: types.TransactionType.TransferAlgo,
       sign: types.SignType.SecretKey,
       fromAccount: master,
-      toAccountAddr: account.addr,
+      toAccountAddr: a.addr,
       amountMicroAlgos: 20e6,
       payFlags: { totalFee: 1000, note: 'funding account' }
     });
+  }
```
Comment (ratik21) [inline examples/permissioned-token/scripts/common/common.js:33]: here we're pushing in `account` array while looping  on account.

Comment (ratik21) [reply examples/permissioned-token/scripts/common/common.js:33]: this should be `params.push`. Right?

Comment (robert-zaremba) [reply examples/permissioned-token/scripts/common/common.js:33]: yes, need to run all of this. Updated. Thanks for spotting.

File: examples/permissioned-token/scripts/user/transfer.js [hunk 2]
```diff
@@ -114,35 +114,33 @@ async function run (runtimeEnv, deployer) {
...
-  ]);
+  /* Fund john, bob, permissionsManager accounts by master **/
+  await fundAccount(deployer, [permissionsManager, john, bob, elon]);
 
   // opt-in accounts to permissions smart contract
   // comment this code if already opted-in
-  await optInAccountToSSC(deployer, elon, permissionsSSCInfo.appID, {}, {});
-  await optInAccountToSSC(deployer, bob, permissionsSSCInfo.appID, {}, {});
-  await optInAccountToSSC(deployer, john, permissionsSSCInfo.appID, {}, {});
+  await Promis.all([
```
Comment (ratik21) [inline examples/permissioned-token/scripts/user/transfer.js:122]:
```suggestion
  await Promise.all([
```

File: examples/permissioned-token/scripts/user/transfer.js [hunk 3]
```diff
@@ -114,35 +114,33 @@ async function run (runtimeEnv, deployer) {
...
+    optInAccountToSSC(deployer, elon, permissionsSSCInfo.appID, {}, {}),
+    optInAccountToSSC(deployer, bob, permissionsSSCInfo.appID, {}, {}),
+    optInAccountToSSC(deployer, john, permissionsSSCInfo.appID, {}, {}),
```
Comment (ratik21) [inline examples/permissioned-token/scripts/user/transfer.js:125]:
```suggestion
    optInAccountToSSC(deployer, john, permissionsSSCInfo.appID, {}, {})
```

**Review by ratik21 — APPROVED** [review]
lgtm, had left two suggestions.

**Pull request merged**